pkg/unifiedpush/client.go

```diff
@@ -335,6 +344,97 @@ func (c UnifiedpushClient) DeleteIOSVariant(v *pushv1alpha1.IOSVariant) error {
 	return nil
 }
 
+// GetWebPushVariant does a GET for a given Variant based on the VariantId
+func (c UnifiedpushClient) GetWebPushVariant(v *pushv1alpha1.WebPushVariant) (WebPushVariant, error) {
+	id := ""
```

Comment:
Can you say `id string` here instead?

Reply:
I think it would have to be as-is, or `var id string`, not `id string` or `id string := ""`

I'm fine with it as-is, I think it's probably more common for declaration & initialization of variables inside a function.